src/main/java/org/kohsuke/github/GHIssue.java

```diff
@@ -198,6 +204,15 @@ public GHIssueState getState() {
         return Enum.valueOf(GHIssueState.class, state.toUpperCase(Locale.ENGLISH));
     }
 
+    /**
+     * Gets state reason.
+     *
+     * @return the state reason
+     */
+    public GHIssueStateReason getStateReason() {
+        return EnumUtils.getEnumOrDefault(GHIssueStateReason.class, state_reason, GHIssueStateReason.UNKNOWN);
```

Comment:
Are you sure this can't be null? My understanding is that when the issue is not closed, it will be null but I might be mistaken.
If so, please use `EnumUtils.getNullableEnumOrDefault()`. I pointed you to it in my previous comment but it might have been a bit confusing as I pointed to this one first.

Reply:
Updated, I was kinda thinking it should be set to UNKNOWN when null, but obviously null makes more sense ;)